components/Workshop/WorkshopListingPastSlotList.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { Box, Heading, Stack } from "@chakra-ui/react";
+import { Booking, Slot } from "@schemas";
+import { WorkshopListingPastSlot } from "@components/Workshop/WorkshopListingPastSlot";
+import Show from "@components/Helpers/Show";
+import NoResults from "@components/NoResults";
+
+interface IProps {
+  slots: Slot[];
+  bookings: Booking[];
+}
+
+export default function WorkshopListingPastSlotList({
+  slots,
+  bookings,
+}: IProps) {
+
+
+  const getActiveBookingsForSlot = (slot: Slot): Booking[] => {
+    return bookings.filter((b) => b.slot_id === slot.id);
+  };
+
+
+  return (
```

Comment:
I am wondering whether it makes sense to not display this component at all, if there are no slots in the past. 
An empty list doesn't add any value to the user in my opinion, but I am happy to be convinced otherwise

Reply:
I understand your point and I agree, there is no need to show that component if there aren't any past sessions. Since we are talking about displaying these lists in tabs, I can implement both versions and maybe we can decide based on which looks better

Reply:
If we go with tabs then we can defintely keep both, it will probably be confusing otherwise

Reply:
![image](https://github.com/user-attachments/assets/379eb4bb-5e8e-427c-b2ff-f987e2c60144)
![image](https://github.com/user-attachments/assets/ba68b399-6952-4130-afdc-94660d05a617)

What do you think about this look for tabs?

Reply:
They look sick :+1: